modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -57,6 +57,9 @@ It is highly recommended to keep it enabled and not to alter it. +
 If you require the _debug.log_ messages in a different format, create an additional Appender since the Neo4j Support team uses the default one.
 For more information, see xref:monitoring/logging.adoc#_default_logging_configuration[Default logging configuration].
 
+Starting from 2025.05, the `failureReason` column of the JSON format query log is deprecated.
```

Comment:
I think we need to move this info under the Deprecation section in the page. To create there a new subsection dedicated to Logging.

Reply:
Oh thanks, I did that first and then I moved it when I saw logging without realising I was no longer under deprecation